webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -186,6 +189,16 @@ class SubscriptionsPage extends Component {
 
     const csvParams = createSubscriptionParams({ search: this.state.searchQuery });
 
+    const emptyStateData = {
```

Comment:
Had to pull this back into the subscriptions page component in order to open the manifest.  Open to other solutions.

Reply:
@tstrachota especially interested in your opinion on this since you did the initial refactor into the subscriptions table component.

Reply:
I see two solutions here:
1. The `EmptyState` component currently takes prop action: `{url, title}`. We can change it to `{url, title, onClick}` and let the component decide whether to wrap the button with `LinkContainer` (when `url` is present) or just render a button with `onClick`.
1. Alternatively you could keep `EmptyState` as is in this PR and change the empty state date to a function and inject the handler:
```js
const emptyStateData = ({onButtonClick}) => ({
  header: __('There are no Subscriptions to display'),
  description: __('Import a Manifest to manage your Entitlements.'),
  actionButton: (
    <Button onClick={onButtonClick}>
      {__('Import a Manifest')}
    </Button>
  ),
}};
```
Both solutions seem fine to me. I slightly tend to prefer the first one. Adding `actionButton` prop feels like adding another way to achieve the same to me.

Reply:
@tstrachota updated.